tests/emit.rs

```diff
@@ -1,6 +1,6 @@
 extern crate edn_rs;
 
-#[cfg(test)]
+#[cfg(feature = "json")]
```

Comment:
Shouldn't it be both `json` feature and `test`?
We're probably compiling it even when its not test compilation.

Reply:
test dir doesn't compile